src/vs/workbench/contrib/mergeEditor/browser/mergeEditorInputModel.ts

```diff
@@ -428,63 +435,56 @@ class WorkspaceMergeEditorInputModel extends EditorModel implements IMergeEditor
 			const message = isMany
```

Comment:
@hediet fyi please check, this one is a bit more complex adoption of the new dialog api